wire/towire.c

```diff
@@ -257,6 +257,16 @@ void towire_bitcoin_tx_output(u8 **pptr, const struct bitcoin_tx_output *output)
 	towire_u8_array(pptr, output->script, tal_count(output->script));
 }
 
+void towire_witscript(u8 **pptr, const struct witscript *script)
+{
+	if (script == NULL || script->ptr == NULL) {
+		towire_u16(pptr, 0);
+	} else {
```

Comment:
Subtly, the script->ptr check here is redundant (as `tal_count(NULL)` == 0), though I'd be tempted to assert it's set, once the previous change is done so it's never NULL.

Reply:
Fixed.